prd.md

```
@@ -43,6 +43,8 @@ Below is the proposed information architecture / interface for the CLI, which ma
 
 #### The following commands are scoped to the account and depend on the control API access token for authentication
 
+*Note: The `topicSeparator` configured for oclif is not a `:` but instead a space (` `). All commands must respect this configuration*
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)